### ISSUE TITLE
Misspelled word is not marked inside of method

### ISSUE THREAD

**Issue by bostick** [streetsidesoftware/vscode-spell-checker#2715]
The Java code:
```java
public void setJournallSizeLimit(int limit) {
    a.setJournallSizeLimit(limit);
}
```

is marked as:
<img width="470" alt="Screenshot 2023-07-28 at 1 43 53 PM" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/183619/b1ddf196-5580-435a-95a7-8f0f7f06615a">

The first misspelled "Journall" is correctly marked, but the second misspelled "Journall" is not marked.

If this is not a bug, then please explain how to enable the second "Journall" to marked as misspelled.

**Comment by Jason3S** [comment]
@bostick,

There is an ignore regular express that tells the spell checker to ignore words after a `.`.

https://github.com/streetsidesoftware/cspell-dicts/blob/fd54c375d59ba84c02b3d381d36a78cdc2d8e7c7/dictionaries/java/cspell-ext.json#L20-L22

<img width="719" alt="image" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/6b82b915-1ab0-483d-aaf3-9b5f48a60c7f">

**Comment by bostick** [comment]
Is ignoring member functions correct behavior?

For example, my misspelled word was not marked because the class definition for the `a` object was not a source file in my project.

I don't have the source for the class of the `a` object in the project and there is nothing telling me about the spelling error.

Is there a way to override this behavior and allow the second "Journall" to marked as misspelled?

I see how to add words to dictionaries, but I could find any reference for how to override this behavior.

**Comment by Jason3S** [comment]
@bostick,

It is possible to override patterns locally by defining one locally with the same name.

This can be done using a CSpell config file:

`cspell.config.yaml`
```yaml
---
$schema: https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json
patterns:
  - name: java-member-function
    pattern: ""
```

Or in the VS Code settings:

<img width="1042" alt="image" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/7786f585-0cdf-4b4a-8aef-2d8b37a4ede5">

## Logic

Often a developer doesn't have control over spelling issues in 3rd party libraries / imports. So, ignoring member functions is a trade-off. It is assumed that the Java compiler will catch incorrectly typing the name of the member function. If it is a locally defined function, then the spell checker will pick it up at the point of definition.

**Comment by bostick** [comment]
Understand about the trade-off. I've added this to my settings and I see the member function now being marked. Thank you!

<img width="449" alt="Screenshot 2023-07-31 at 12 51 51 PM" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/183619/c0fc3cec-92e5-4a8a-8818-313fc1d3ea0b">
